derive/src/rpc_attr.rs

```diff
@@ -162,6 +189,11 @@ fn validate_attribute_meta(meta: syn::Meta) -> Result<syn::Meta> {
 			validate_idents(&meta, &visitor.name_value_names, &[SUBSCRIPTION_NAME_KEY, RPC_NAME_KEY])?;
 			validate_idents(&meta, &visitor.meta_list_names, &[ALIASES_KEY])
 		}
+		NOTIFICATION_ATTR_NAME => {
```

Comment:
Same here, I don't think we use it any more, since we use the output type now.

Reply:
Good catch! I'll remove the unused attribute from the code right away.